libs/blocks/quiz/quizoption.js

```diff
@@ -1,40 +1,62 @@
 import { html } from '../../deps/htm-preact.js';
 
 export const OptionCard = ({
-  text, title, coverImage, cardIcon, options, disabled, selected,
+  text, title, image, icon, iconTablet, iconDesktop, options, disabled, selected, background,
 }) => {
-  const cardIconHtml = html`<div class="quiz-option-icon">
-    <img src="${cardIcon}" alt="Quiz Option Icon" />
+  const getOptionClass = () => {
+    let className = '';
+    if (icon || iconTablet || iconDesktop) className += 'has-icon ';
+    if (image) className += 'has-image ';
+    if (background) className += 'has-background ';
+    if (disabled) className += disabled;
+    if (selected) className += selected;
+    return className;
+  };
+
+  const getIconHtml = (iconUrl, iconClass) => html`<div class="quiz-option-icon ${iconClass}">
+    <img src="${iconUrl}" alt="Quiz Option Icon" />
```

Comment:
"Quiz Option Icon" - can you get the alt from the image or do authors have to do the replace thing for loc?

Reply:
We should likely set alt text to the card title or description, rather than something hardcoded like this value.

Reply:
Good call on these. I hadn't touched alt tags at all in all of this,  but we should probably at least add some checks for title, or description before defaulting to the current generic text.

Reply:
Set this to use text or title before resorting to the generic default